core/src/constants.rs

```diff
@@ -0,0 +1,36 @@
+// This file is part of Gear.
+
+// Copyright (C) 2021-2025 Gear Technologies Inc.
+// SPDX-License-Identifier: GPL-3.0-or-later WITH Classpath-exception-2.0
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+//! Runtime constants which we want to keep in one place instead
+//! of changing them in multiple places.
+
+/// Limit of outgoing messages per block.
```

Comment:
could you organize them properly here: sorted + with doc headers for related things
sth like this for ease of use and read

```rust
// # Messaging constants #

// Im a 
a 

// Im b
b

// # Runtime constants #

// Im c
c 

```